extension/fluentbitextension/README.md

```diff
@@ -1,5 +1,11 @@
 # Deprecated FluentBit Subprocess Extension
 
+| Status                   |                      |
+| ------------------------ |----------------------|
+| Stability                | [deprecated]         |
+| Supported pipeline types | trace, metrics, logs |
```

Comment:
I don't think we should put a list here, extensions don't handle any signal type

```suggestion
| Supported pipeline types | n/a |
```